src/include/distributed/worker_transaction.h

```diff
@@ -29,11 +29,22 @@ typedef enum TargetWorkerSet
 	 */
 	NON_COORDINATOR_METADATA_NODES,
 
+	/*
+	 * All the active primary nodes in the metadata which have metadata
+	 * except the local node
+	 */
+	REMOTE_METADATA_NODES,
+
 	/*
 	 * All the active primary nodes in the metadata except the coordinator
 	 */
 	NON_COORDINATOR_NODES,
 
+	/*
+	 * All the active primary nodes in the metadata except the local node
+	 */
+	REMOTE_NODES,
```

Comment:
In #7203 this is called OTHER_NODES. I'm not sure which name I like best @halilozanakgul @onurctirtir what do you think?

Reply:
@thanodnl responded in private chat:
> REMOTE_NODES > OTHER_NODES

Reply:
`REMOTE_NODES` > `OTHER_NODES` imho

Reply:
OTHER_NODES sounds good to me too

Reply:
As discussed over chat, lets go for REMOTE_NODES